axes/__init__.py

```diff
@@ -1,4 +1,12 @@
-from pkg_resources import get_distribution
+def get_axes_version():
```

Comment:
Could we have this as plain top-level code without a function wrapping it?

The style checker disagrees with non-top-level imports and the QA test run won't hence pass.